x/stakeibc/types/message_liquid_stake.go

```diff
@@ -32,6 +32,14 @@ func StAssetDenomFromHostZoneDenom(hostZoneDenom string) string {
 	return "st" + hostZoneDenom
 }
 
+func IsStAssetDenom(stAssetDenom string) bool {
+	return strings.HasPrefix(stAssetDenom, "st")
+}
+
+func HostZoneDenomFromStAssetDenom(stAssetDenom string) string {
+	return stAssetDenom[2:]
+}
+
```

Comment:
nit: this feels like a weird place to put these helpers. Might make more sense to put them in `types/host_zone.go`

Reply:
agreed, moved

Reply:
you may have forgotten to push up this commit?

Reply:
just moved, resolving